src/main/java/com/zotov/edu/passportofficerestservice/repository/PassportsRepositoryCollections.java

```diff
@@ -14,67 +18,74 @@
 import java.util.stream.Collectors;
 
 @Repository
+@AllArgsConstructor
 public class PassportsRepositoryCollections implements PassportsRepository {
```

Comment:
to manage which bean is is context: memory or jdbc one one can use `@ConditionalOnProperty` like that:
```
@ConditionalOnProperty(value = "app.repository", havingValue = "memory")
```

Reply:
@ConditionalOnProperty has been added to the repositories implementations